ingress/ingress_test.go

```diff
@@ -24,6 +26,16 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+const StableService string = "test-stable-service"
+const AdditionalStableService string = "test-stable-service-additional"
```

Comment:
Is this used?

Reply:
no, I believe this was missed cleanup when I was changing tests based off of the change to have `stableIngress`/`stableIngresses` be exclusive fields. thanks!